### ISSUE TITLE
The current date picker has no 'Now' button as stated in the information message when editing a flash message

### ISSUE THREAD

**Issue by ilieandriuta** [xwikisas/application-flashmessages#47]
When creating a new flash message entry or edit an existing one, the information message displayed on the page states that `to check the UTC time you can use the button 'now' in the calendar`.

However, the new date picker has no 'Now' button like the old picker.
As a result, this information affects usability as it's confusing for the users and the info message should be improved.

![Flash_no_Now_button](https://user-images.githubusercontent.com/32512703/76529251-b85b4b80-647a-11ea-91aa-777cbc1ea395.png)

Environment: Edge 80, using an instance of XWiki 12.1 on PostgreSQL 12, Docker

**Comment by acotiuga** [comment]
Indeed, the date picker was improved in the latest versions of XWiki and the `Now` button is not present anymore. We should analyze if there is another way to provide this UTC time checking or even if it brings any value since is the single place in the whole XWiki and there are many other places with date pickers.
